ajax/libs/react-block-ui/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "react-block-ui",
+  "filename": "reactblockui.min.js",
+  "description": "Block UI for react",
+  "homepage": "https://availity.github.io/react-block-ui/",
+  "keywords": [
+    "availity",
+    "loader",
+    "blockui",
+    "redux",
+    "react",
+    "component",
+    "components",
+    "react-component",
+    "ui"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/Availity/react-block-ui.git"
```

Comment:
Can we follow the upstream which will be https://github.com/Availity/react-block-ui.git?